pkg/tfbridge/provider.go

```diff
@@ -298,22 +298,36 @@ func (p *Provider) label() string {
 
 // initResourceMaps creates maps from Pulumi types and tokens to Terraform resource type.
 func (p *Provider) initResourceMaps() {
+
+	ignoredTokens := ignoredTokens(&p.info)
+
 	// Fetch a list of all resource types handled by this provider and make a map.
 	p.resources = make(map[tokens.Type]Resource)
 	p.tf.ResourcesMap().Range(func(name string, res shim.Resource) bool {
+		schema, ok := p.info.Resources[name]
```

Comment:
We checked for p.info.Resources == nil before - are you allowed to call [name] on a nil map?

Reply:
You are allowed to treat a nil map as indexable, if it is an rvalue:

```
x := mayBeNil[name] // This is ok, x is the zero value

mayBeNil[name] = x // This will panic
```